nix/kexec-installer/kexec-run.sh

```diff
@@ -65,7 +65,7 @@ if printf "%s\n" "6.1" "$(uname -r)" | sort -c -V 2>&1; then
   kexecSyscallFlags="--kexec-syscall-auto"
 fi
 
-if ! "$SCRIPT_DIR/kexec" --load "$SCRIPT_DIR/bzImage" \
+if ! "$SCRIPT_DIR/kexec" --no-sync --load "$SCRIPT_DIR/bzImage" \
```

Comment:
Do we actually need to care about this in the load command?
Afaik this only should have an effect in the actual exec step?

```suggestion
if ! "$SCRIPT_DIR/kexec" --load "$SCRIPT_DIR/bzImage" \
```

Reply:
When testing I found it to also get stuck in the load call. Had exactly the same assumption initially 😄

Reply:
Ok. I guess we can keep it here than anyway.